### PR TITLE
Add Ethernity Mainnet

### PR DESCRIPTION
**Pull request by denis-pingin**
# Adding a New Chain
This PR adds [Chain Name Here] to the registry.

## Checklist
- [x] I have declared the chain at the appropriate [Superchain Level](../docs/glossary.md#superchain-level-and-rollup-stage).
- [x] I have run `just validate <chain-id>` locally to ensure all local validation checks pass. 
- [x] I have run `just codegen` to ensure that the chainlist and other generated files are up-to-date and include my chain.

### CODE REVIEW

File: validation/genesis/validation-inputs/183/meta.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+genesis_creation_commit = "9047beb54c66a5c572784efec8984f259302ec92"
+node_version = "18.12.1"
+monorepo_build_command = "pnpm"
+genesis_creation_command = "opnode2"
```
Comment (bitwiseguy) [inline validation/genesis/validation-inputs/183/meta.toml:4]: Please change this from `opnode2` to `forge1`. This matches the command used for the Ethernity testnet and should get the `just validate-genesis-allocs 183` ci test to pass

File: chainList.json [hunk 2]
```diff
@@ -357,4 +375,4 @@
       "chain": "sepolia-dev-0"
     }
   }
-]
```
Comment (bitwiseguy) [inline chainList.json:360]: This edit is causing the codegen ci test to fail. Please run `just codegen` again and push the change

File: validation/genesis/validation-inputs/183/meta.toml (hunk 1 above)
Comment (denis-pingin) [reply validation/genesis/validation-inputs/183/meta.toml]: Updated, `just validate-genesis-allocs 183` passed locally

File: chainList.json (hunk 2 above)
Comment (denis-pingin) [reply chainList.json]: Fixed. This change was caused by this line in .editorconfig, it seems to be inconsistent with the CI pipeline: https://github.com/ethereum-optimism/superchain-registry/blob/ea7e768a02a6f88a38a1d39d27b9be6c212f4b6e/.editorconfig#L9

**Review by sbvegan — APPROVED** [review]

**Review by bitwiseguy — APPROVED** [review]

**Pull request merged**